common/saber_base.h

```diff
@@ -194,6 +196,7 @@ public:                                                         \
   static void DoClash() { DoEffectR(EFFECT_CLASH); }
   static void DoBlast() { DoEffectR(EFFECT_BLAST); }
   static void DoForce() { DoEffectR(EFFECT_FORCE); }
+  static void DoMonoForce() { DoEffectR(EFFECT_MONOFORCE); }
```

Comment:
why does MONOFORCE get a Do* function but not SWAP?

Reply:
I think because MONOFORCE can do a blade effect and sound, while SWAP is just event used as a trigger with a sound.